lib/ansible/modules/cloud/amazon/lambda.py

```diff
@@ -282,33 +303,24 @@ def main():
     check_mode = module.check_mode
     changed = False
 
-    if not HAS_BOTOCORE:
-        module.fail_json(msg='Python module "botocore" is missing, please install it')
-
-    if not HAS_BOTO3:
-        module.fail_json(msg='Python module "boto3" is missing, please install it')
-
-    region, ec2_url, aws_connect_kwargs = get_aws_connection_info(module, boto3=True)
+    region, ec2_url, aws_connect_kwargs = get_aws_connection_info(
+        module, boto3=True)
```

Comment:
You fixed a lot of these, thanks. This one still needs fixing. I'll comment on the few others that don't seem to add to the readability for your ease of reference.